### ISSUE TITLE
Do we need `watchexec` listed as a buildpack?

### ISSUE THREAD

**Issue by ryanmoran** [paketo-buildpacks/full-builder#684]
## Expected Behavior

I didn't expect to find `watchexec` listed as a top-level buildpack in the builder, but there's this:
https://github.com/paketo-buildpacks/full-builder/blob/2b4e15f049c9a57bd383878d9a9ec3ccaf7bbea2/builder.toml#L39-L41

## Current Behavior

I think this means we just have this unnecessary reference.

## Possible Solution

Remove the lines related to `watchexec`.

## Steps to Reproduce

1. visit https://github.com/paketo-buildpacks/full-builder/blob/2b4e15f049c9a57bd383878d9a9ec3ccaf7bbea2/builder.toml#L39-L41

## Motivations

I just want to make sure we aren't pulling in extra copies of buildpacks that we don't ultimately use.

**Comment by fg-j** [comment]
I agree this is superfluous. Given it's not included in any orders within the builder, I'm not sure how a user would even access the buildpack. This looks like a vestige of when watchexec + nginx [appeared together in an order group](https://github.com/paketo-buildpacks/full-builder/commit/eb7efb167ebad9ccb520bdefc234718d914f265d) before the web-servers buildpack was complete.
